src/NServiceBus.Core.Tests/Sagas/ComponentTests/AnotherSagaWithCorrelatedPropertyData.cs

```diff
@@ -0,0 +1,28 @@
+namespace NServiceBus.Persistence.ComponentTests
+{
+    using System;
+    using System.Threading.Tasks;
+
+    class AnotherSagaWithCorrelatedProperty : Saga<AnotherSagaWithCorrelatedPropertyData>, IAmStartedByMessages<TwoUniquePropertyStartingMessage>
+    {
+        public Task Handle(TwoUniquePropertyStartingMessage message, IMessageHandlerContext context)
+        {
+            throw new NotImplementedException();
+        }
+
+        protected override void ConfigureHowToFindSaga(SagaPropertyMapper<AnotherSagaWithCorrelatedPropertyData> mapper)
+        {
+            mapper.ConfigureMapping<TwoUniquePropertyStartingMessage>(m => m.CorrelatedProperty).ToSaga(s => s.CorrelatedProperty);
+        }
+    }
+
+    class TwoUniquePropertyStartingMessage
```

Comment:
no idea. but this was not introduced in this PR so should be raised on a diff issue